public/app/features/dashboard-scene/scene/NavToolbarActions.tsx

```diff
@@ -181,7 +182,7 @@ export function ToolbarActions({ dashboard }: Props) {
     });
     leftActions.push({
       group: 'add-panel',
-      condition: isEditingAndShowingDashboard,
+      condition: isEditingAndShowingDashboard && dashboard.state.body instanceof RowsLayoutManager,
```

Comment:
we should always support this action, can add it to the top row (or bottom row)

Reply:
Fixed. For responsive grid we're switching to rows layout in case the user clicks add row. Nothing is changed for default grid.